torch/_torch_docs.py

```diff
@@ -4610,11 +4610,11 @@ def merge_dicts(*dicts):
 Returns the q-th quantiles of each row of the :attr:`input` tensor along the dimension
 :attr:`dim`, doing a linear interpolation when the q-th quantile lies between two
 data points. By default, :attr:`dim` is `None` resulting in the :attr:`input` tensor
-beign flattened before computation.
+being flattened before computation.
 
 If :attr:`q` is a 1D tensor, the first dimension of the result corresponds to the quantiles
```

Comment:
"corresponds to the quantiles" and "what remains from the reduction" are vague expressions. Can we be more precise here?